src/signals_notebook/entities/users/profile.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
+from typing import List, Literal, Union
+
+from pydantic import BaseModel, Field
+
+from signals_notebook.common_types import ObjectType, Response
+
+
+class Role(BaseModel):
+    id: int
+    name: str
+
+
+class Licence(BaseModel):
+    id: Union[int, str]
+    name: str
+    expires_at: datetime = Field(alias='expiresAt')
+    valid: bool
+    has_service_expired: bool = Field(alias='hasServiceExpired')
+    has_user_found: bool = Field(alias='hasUserFound')
+    has_user_activated: bool = Field(alias='hasUserActivated')
+
+    class Config:
+        validate_assignment = True
+        allow_population_by_field_name = True
+
+
+class Profile(BaseModel):
+    id: str = Field(alias='userId', allow_mutation=False)
+    first_name: str = Field(alias='firstName')
+    last_name: str = Field(alias='lastName')
+    email: str
+    created_at: datetime = Field(alias='createdAt', allow_mutation=False)
+    tenant: str
+    roles: List[Role]
+    licenses: List[Licence]
+
+    class Config:
+        validate_assignment = True
+        allow_population_by_field_name = True
+
+
+class ProfileResponse(Response[Profile]):
+    pass
+
+
+class Group(BaseModel):
+    type: Literal[ObjectType.GROUP] = Field(allow_mutation=False)
```

Comment:
When I created this class, I could not understood this thing: Why cannot we use EntityType.Group? I had no idea how to solve this problem.

Reply:
It could be better use EntityType (not ObjectType). But in this case code didnt work for me. Maybe try to change it now?